supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -27,3 +27,29 @@ Provide the source language if it is supported by the Red Hat Customer Portal to
 == Long code examples in procedures
 
 All code blocks (regardless of length) must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts. The length of the block is irrelevant, within reason, if the code block follows these guidelines.
+
+[[commands-with-elevated-privileges]]
+== Commands with elevated privileges
+
+Some commands require elevated privileges to run. Users without elevated privileges must first do one of the following to run such a command:
+
+* Use `su -i` to switch to the root user account.
+* Preface the command with `sudo` to temporarily change their current privileges.
+
+=== Guidelines for documenting commands with elevated privileges
+
+Use the following guidelines when you document commands that require elevated privileges:
+
+* If you include a shell prompt in a sample command, always show the correct prompt for a regular user (`$`) or a user with elevated privileges (`#`).
++
+[NOTE]
+====
+Do not rely solely on a shell prompt in a sample command to indicate the required privilege level to run a command.
```

Comment:
Should we continue this statement with a suggestion that if you are using # to indicate that the root user is required, that there should also be a statement in the step text (or prereqs)?

Reply:
Good idea. Added a sentence. Lmk what you think,